packages/backend/src/accounting/service.ts

```diff
@@ -8,6 +8,14 @@ export enum LiquidityAccountType {
   WEB_MONETIZATION = 'WEB_MONETIZATION'
 }
 
+export interface SettlementAccount {
+  id: string
+  asset: {
+    id: string
+    ledger: number
+  }
+}
```

Comment:
Why are there two `id`s and isn't the id not just the ledger as string?

Reply:
🤔 yeah I think that's just better & simpler. I think originally I liked that [this](https://github.com/interledger/rafiki/pull/1095/files#diff-db7f903acbbd222c73bf85a026fc87e5dfb271fede799b42a5a163e8bc62fd29R69-R73) was explicit in showing that accountRef = assetId for settlement accounts, but it'll be just as clear on the caller side with a single `id: string` argument
```
  await deps.ledgerAccountService.create({
    accountRef: account.asset.id,
    assetId: account.asset.id,
    type: LedgerAccountType.SETTLEMENT
  })
```

Reply:
Oh, I know why now, one moment

Reply:
I think I prefer `getSettlementAccount` taking in a `SettlementAccount` argument, thoughts?